### DISCUSSION TITLE
Use multiple GO terms or just one?

### DISCUSSION THREAD

**Discussion by aspencoyle** [RobertsLab/resources#1380] [Q&A] [answered]
Hi all,

I've been poking around the Tanner crab transcriptome, and I noticed something.

Previously, I'd been examining genes with the GO term for "immune response" (n = 87). However, substantially more genes have the GO term for "innate immune response" (n = 428).

I've done quite a bit of research on the 87 immune response genes, so I'd prefer not to throw those out. However, it really seems appropriate to include those 400-odd other immune-related genes.

I know that generally, grouping together genes from a whole bunch of different GO terms is frowned upon. But in this case, would it be acceptable for my methods section to include something like

`The cross-referenced table of accession IDs and GO terms for each transcriptome was filtered to examine genes with the GO term for "Immune Response" (GO: 0006955) and "Innate Immune Response" (GO:0045087). Literature searches for function in closely related species were then performed
`

Or should I restrict myself to only including genes matching a single specific GO term?

Thank you!

**Comment by kubu4** [comment]
I don't have a response to your question, but just wanted to clarify this:

> I know that generally, grouping together genes from a whole bunch of different GO terms is frowned upon.

This isn't frowned upon, as this is essentially what a GOslims do (group related GO terms into a higher level group), which in turn ends up grouping the genes.

**Reply by aspencoyle** [reply]
Sounds good - I suppose what I mean is grouping them together more arbitrarily isn't acceptable methodology for a paper (such as, say, including all GO terms with "immune" somewhere in the name)

**Reply by kubu4** [reply]
Well, as it turns out, that is _also_ acceptable. That type of grouping is called "semantic similarity", where GO terms a grouped based off of how the _words_ in a GO terms are related.

**Reply by aspencoyle** [reply]
Well holy moly, thanks a bunch - good to know!

**Comment by sr320** [comment]
> Previously, I'd been examining genes with the GO term for "immune response" (n = 87). However, substantially more genes have the GO term for "innate immune response" (n = 428).

I presume this does not involve the discussion of genes determined to be influenced by temperature or time as we searched for the word "immune"?

**Reply by aspencoyle** [reply]
Nope, this is for the general description of immune-related genes within the transcriptome

**Reply by sr320** [reply]
So finish up differential gene expression aspect first.. then we can move back to address general description as this is secondary.

**Reply by aspencoyle** [reply]
Alright, think I've finished a pretty good description of differential gene expression (or made some substantial progress - some room left for improvement in the parasite differential expression analysis, and a round of edits needed throughout, but the gist is there)

**Comment by sr320** [comment, marked as answer]
As a more general comment here, it is important to remember GO is hierarchal and it is good idea to leverage ancestor/parent (GOslim) terms to capture relevant genes


<img width="514" alt="Screen Shot on 2022-02-02 at 13-32-37" src="https://user-images.githubusercontent.com/4442039/152240880-1243fdb1-1009-4db4-a614-de442f3026f8.png">
